cmake/CMakeLists.txt

```diff
@@ -1003,32 +1035,108 @@ if (CPUINFO_SUPPORTED)
     string(APPEND CMAKE_CXX_FLAGS " -DCPUINFO_SUPPORTED")
   endif()
 
-  if (onnxruntime_PREFER_SYSTEM_LIB)
-    find_package(cpuinfo)
-  endif()
-  if (cpuinfo_FOUND)
-    message("Use cpuinfo from preinstalled system lib")
-  else()
-    message("Use cpuinfo from submodule")
-    set(CPUINFO_BUILD_TOOLS OFF CACHE INTERNAL "")
-    set(CPUINFO_BUILD_UNIT_TESTS OFF CACHE INTERNAL "")
-    set(CPUINFO_BUILD_MOCK_TESTS OFF CACHE INTERNAL "")
-    set(CPUINFO_BUILD_BENCHMARKS OFF CACHE INTERNAL "")
-    add_subdirectory(external/pytorch_cpuinfo EXCLUDE_FROM_ALL)
-    if (MSVC)
-      target_compile_options(cpuinfo PRIVATE "-D_CRT_SECURE_NO_WARNINGS")
```

Comment:
We may still need it?

Reply:
We don't treat their warnings as errors. So we don't need to suppress their warnings. If we feel them annoying, we should submit a PR to cpuinfo's repo.